docs/ldml/tr35-info.md

```diff
@@ -1116,87 +1116,110 @@ The examples in #4 are due to the following ordering of the `unitQuantity` eleme
 
 ## Mixed Units
 
-Mixed units, or unit sequences, are units with the same base unit which are listed in sequence. Common examples are feet and inches, meters and centimeters, and hours, minutes, and seconds. Mixed unit identifiers are expressed using the "-and-" infix, as in "foot-and-inch", "meter-and-centimeter", and "hour-and-minute-and-second".
+Mixed units, or unit sequences, are units with the same base unit which are listed in sequence. 
+Common examples are feet and inches, meters and centimeters, and hours, minutes, and seconds. 
+Mixed unit identifiers are expressed using the "-and-" infix, as in "foot-and-inch", "meter-and-centimeter", and "hour-and-minute-and-second".
 
-Scalar values for mixed units are expressed in the largest unit, according to the sort order discussed above in "Normalization". For example, numbers for "foot-and-inch" are expressed in feet.
+Scalar values for mixed units are expressed in the largest unit, according to the sort order discussed above in "Normalization". 
+For example, numbers for "foot-and-inch" are expressed in feet.
 
-Mixed units are expected to be rendered in the order of the tokens in the identifier. For example, the value 1.25 with the identifier "foot-and-inch" should be rendered as "1 foot and 3 inches" and 1.25 inch-and-foot should be rendered as “3 inches and 1 foot". **NOTE:** the correct application of this may require adding locales to the regions attribute set.
+Mixed unit identifiers should be from largest to smallest (eg foot-and-inch instead of inch-and-foot). 
+If it turns out that some locales present certain mixed units in a different order, additional structure will be needed in CLDR.
+
+If a maximum/minimum number of decimals or signficant digits is desired in formatting, 
+those values should should be adapted to the mixed radix used, starting with the largest unit, 
+and discarding smaller units where they would be overly accurate. 
+For example, if the goal is 1 part in 100, then "6 miles 1 foot" is overly precise, 
+while "6 foot 1 inch" has approximately the right accuracy.
```

Comment:
I think it would be much clearer and easier to implement if the decimal formatting options were applied to the smallest unit, with larger units always displayed as an integer.

This appears to be what you write farther down in the doc:

> * If the unit is mixed (eg foot-and-inch) the skeleton applies to the final subunit; the higher subunits are formatted as integers.

Also consider the case "1.9999999... ft" formatted with foot-and-inch. This should result in "2 feet", not "1 foot 12 inches".

Reply:
First, when I wrote "while "6 foot 1 inch" has approximately the right accuracy." I had flubbed the example. So disregard that bit.

> I think it would be much clearer and easier to implement if the decimal formatting options were applied to the smallest unit, with larger units always displayed as an integer.

I agree it is simpler for us to implement. And we're in agreement on the higher units always showing as integers.
But it gives less satisfactory results. Take the following example.

The programmer supplies: source unit is meters, and amount is 1616, and desired precision is 3 significant digits. 

For meters, that turns into "1.62 kilometers". That reflects internal values 1.62±0.005 km, or within 5 meters.

In miles, that turns into 1.004135847 decimal. If we take the floor*, we get 1 mile, with 21.83727034 feet remaining. If we apply 3 sig. digits to the last field, then we get "1 mile 21.8 feet". That accuracy is excessive: 21.8±0.05 feet is within 0.015 meters. What we really want in that case is something that reflects the original accuracy: a rounded number that is within 5 meters of the original amount (=~16 feet), which is something like "1 mile 20 feet", not "1 mile 22 feet" or "1 mile 21.8 feet".

\* As you point out, if we get satisfactory precision out of a higher unit, then we don't want to include the lower unit at all. And if that is the case, we would want to use round(x) and not floor(x).

What I was trying to express with the "should" was more of a warning: (a) there is an issue with mixed units, and (b) we don't have a definitive algorithm (yet) for what to do.

Reply:
I understand the desired behavior, but a hand-wavy algorithm can't be implemented. I would rather offer a suggestion of a mechanism that can be revised later than try to hitchhike on an existing mechanism (maximum significant and fractional digits) which are already hard to implement correctly.

> Implementations may offer mechanisms to limit the precision of the formatted mixed unit. For example, "1.62143 meters" with three significant figures could be rendered in meter-and-millimeter as "1 meter, 620 millimeters". This mechanism should be distinct from decimal digit options, such as those provided in a number formatting skeleton, which should be applied to only the smallest unit.

Reply:
Yah, I don't like the handwaviness either. I'll try some wording, but because implementations and (and IMO should) have other mechanisms, we need to not **require** that the precision be applied to the final unit.

Reply:
I'm leery of having yet another precision type for mixed units. I do not think that peopel 

BTW, what I was thinking of having is something like the following (wording is currently clumsy). I'm not going to include this wording, because we don't have time to work out any kinks, but just to give you a sense of what I'm thinking.

1. The original source (say meters) is first converted to the topmost unit, eg miles.
2. The precision is applied to that topmost unit to find an "accuracy" value.
3. Eg, **12.345678 miles** with 3 sig. digits gives **12.3 miles**, so the accuracy is **±0.05 miles**. 
4. As you convert to successively smaller units, stop when the value could be rounded to be within that range.
5. The first unit as an integer is not within that accuracy, so we use the integer value **12 miles**, and the remainder to carry over is **0.345678 miles**
6. The carry-over to feet is **1825.17984 feet**, and accuracy is **±264**. Since the accuracy is > 1, we know we are going to stop at this value, rounding to some amount, and not go on to inches.
7. Rounding to 1 digit (2000 feet) would be within the accuracy, so we stop, and round to that value, producing "12 miles 2000 feet".

This is not by any means complete, just to give you a sense.

Reply:
Please take a look at the current text now, @sffc and @younies .